packages/toolpad-app/src/server/appTemplateDoms/doms.ts

```diff
@@ -0,0 +1,11 @@
+import * as appDom from '../../appDom';
+
+import statisticsAppDom from './statistics.json';
```

Comment:
This will likely not scale well with the typechecking. Can't we read them using `fs` instead?

Reply:
sure, i'll use `fs`